### PR TITLE
Insert cv and param in tracker mode

### PR DESCRIPTION
**Pull request by mikeyford**
#### What does this PR do?
Adds data to patterns using CV input and param values, when in tracker view. Respects settings for IN.SCALE and PARAM.SCALE.

#### How should this be manually tested?
Go into tracker mode, move param knob while pressing `p`. Supply CV at input while pressing `i`

#### Any background context you want to provide?
I was looking for quicker way to create sequences from a keyboard with CV out.

#### I have,
* [X] updated `CHANGELOG.md` and `whats_new.md`
* [X] updated the documentation
* [X] updated `help_mode.c` (if applicable)
* [X] run `make format` on each commit
* [X] run tests

### CODE REVIEW

File: module/help_mode.c [hunk 1]
```diff
@@ -91,7 +91,9 @@ const char* help1[HELP1_LENGTH] = { "1/16 HELP",
                                     "SH-S|SET START",
                                     "SH-E|SET END",
                                     "ALT-L,S,E|JUMP",
-                                    "SHIFT-2|SHOW/HIDE TURTLE" };
+                                    "SHIFT-2|SHOW/HIDE TURTLE",
+                                    "I|SET TO CV INPUT",
+                                    "P|SET TO PARAM KNOB" };
```
Comment (tehn) [inline module/help_mode.c:96]: also need to modify HELP1_LENGTH const

**Review by tehn — CHANGES_REQUESTED** [review]
super cool feature, thanks for adding!

**Comment by tehn** [comment]
not sure how i missed the existing functions, thanks for the heads up

**Comment by scanner-darkly** [comment]
to clarify - currently there is no way to record the input value, just the param value, and i can see the benefit of using PARAM.SCALE in addition to (or maybe as a replacement for?) the predetermined 0..31 / 0..1023 ranges.

**Comment by mikeyford** [comment]
My primary motivation here was entry from CV. I added the param input option to use the `PARAM.SCALE` values mostly since I was here too. It feels more flexible/useful to me than the existing ranges, but I also don't feel at all strongly about it, so happy to drop it if that's preferred.

**Comment by scanner-darkly** [comment]
i think both are useful additions since they offer different functionality!

this does bring up a question of using letter shortcuts in tracker view - so far we don’t have any, and i don’t see why there shouldn’t be some, so the question becomes whether we should add a bunch of others to improve tracker functionality (and maybe make it more of an actual tracker), in which case it would be good to consider what else could be added and allocate letters accordingly.

**Comment by scanner-darkly** [comment]
are you on teletype study discord? we have a dev channel there which is a great place to get quick feedback!

**Review by tehn — APPROVED** [review]

**Comment by tehn** [comment]
agreed with @scanner-darkly --- before introducing letter input to the tracker screen it'd be good to assess letter assignments more broadly

**Comment by scanner-darkly** [comment]
perhaps something like shift-p and shift-i for now?

**Comment by desolationjones** [comment]
I could get behind just changing the existing shortcuts ctrl-alt and ctrl-shift to paste param and in, respectively, scaled to their individual scaling OPs.

Currently:

<html><body>


ctrl-alt | insert knob value scaled to 0..31
-- | --
ctrl-shift | insert knob value scaled to 0..1023


</body>
</html>

Proposed:
<html><body>


ctrl-alt | insert PARAM knob value scaled to `PARAM.SCALE`
-- | --
ctrl-shift | insert `IN` CV value scaled to `IN.SCALE`


</body>
</html>

@mikeyford feel free to pop by the [discord study group](https://discord.gg/WsUAhqSb) to discuss!

**Comment by scanner-darkly** [comment]
i think it's best to keep the existing ones - there might be folks that are used to the old shortcuts, and it would create confusion if people reference the manual but are still on an older version.

**Comment by mikeyford** [comment]
Apologies. Haven't had time to follow up on this yet, and I'm away from my Teletype for the next few weeks. 

I am planning to join the Discord and discuss this change there once I have time. Also interested/motivated to add further functionality to tracker mode, after collecting some input on the Discord.

**Pull request closed without merging**